components/match2/wikis/heroes/match_summary.lua

```diff
@@ -0,0 +1,484 @@
+---
+-- @Liquipedia
+-- wiki=heroes
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CustomMatchSummary = {}
+
+local Class = require('Module:Class')
+local DisplayHelper = require('Module:MatchGroup/Display/Helper')
+local Flags = require('Module:Flags')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local ChampionIcon = require('Module:HeroIcon')
+local Table = require('Module:Table')
+local ExternalLinks = require('Module:ExternalLinks')
+local String = require('Module:StringUtils')
+local Array = require('Module:Array')
+
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+local MatchSummary = Lua.import('Module:MatchSummary/Base', {requireDevIfEnabled = true})
+
+local MAX_NUM_BANS = 5
+local NUM_CHAMPIONS_PICK = 5
+
+local GREEN_CHECK  = '[[File:GreenCheck.png|14x14px|link=]]'
+local NO_CHECK = '[[File:NoCheck.png|link=]]'
+local MAP_VETO_START = '<b>Start Map Veto</b>'
+local ARROW_LEFT = '[[File:Arrow sans left.svg|15x15px|link=|Left team starts]]'
+local ARROW_RIGHT = '[[File:Arrow sans right.svg|15x15px|link=|Right team starts]]'
+local TBD = 'TBD'
+
+local EPOCH_TIME = '1970-01-01 00:00:00'
+local EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+
+-- Champion Ban Class
+local ChampionBan = Class.new(
+	function(self)
+		self.root = mw.html.create('div'):addClass('brkts-popup-mapveto')
+		self.table = self.root:tag('table')
+			:addClass('wikitable-striped'):addClass('collapsible'):addClass('collapsed')
+		self:createHeader()
+	end
+)
+
+function ChampionBan:createHeader()
+	self.table:tag('tr')
+		:tag('th'):css('width','40%'):wikitext(''):done()
+		:tag('th'):css('width','20%'):wikitext('Bans'):done()
+		:tag('th'):css('width','40%'):wikitext(''):done()
+	return self
+end
+
+function ChampionBan:banRow(banData, gameNumber, numberOfBans, date)
+	self.table:tag('tr')
+		:tag('td')
+			:node(CustomMatchSummary._opponentChampionsDisplay(banData[1], numberOfBans, date, false, true))
+		:tag('td')
+			:node(mw.html.create('div')
+				:wikitext(CustomMatchSummary._createAbbreviation{
```

Comment:
use Module:Abbreviation instead
(also remove the function further down when switching it)